web/src/components/navbar/mobile/NavBarMobileAccountSlideOutMenu.test.tsx

```diff
@@ -0,0 +1,67 @@
+import { NavBarMobileAccountSlideOutMenu } from "@/components/navbar/mobile/NavBarMobileAccountSlideOutMenu";
+import { MenuItem } from "@mui/material";
+import { fireEvent, render, screen } from "@testing-library/react";
+
+describe("<NavBarMobileAccountSlideOutMenu />", () => {
+  window.open = jest.fn();
```

Comment:
posting same comment as desktop 

pebble - we've turned open into a jest function. Any reason we are not resetting mocks after each test. Any reason why we are not reverting the open fn back to what it previously was?